vulture_os/toolkit/api_parser/sentinel_one/sentinel_one.py

```diff
@@ -227,9 +250,12 @@ def execute(self):
 
                 if len(logs) > 0:
                     if event_kind == "alert":
-                        last_timestamp = datetime.fromisoformat(logs[-1]['threatInfo']['updatedAt'].replace("Z", "+00:00"))+timedelta(milliseconds=1)
+                        # timestamps compared by API have a 10ms precision (empiric observation)
+                        last_timestamp = datetime.fromisoformat(logs[-1]['threatInfo']['updatedAt'].replace("Z", "+00:00"))+timedelta(milliseconds=10)
                     else:
-                        last_timestamp = datetime.fromisoformat(logs[-1]['updatedAt'].replace("Z", "+00:00"))+timedelta(milliseconds=1)
```

Comment:
Any reason for removing the `.replace("Z", "+00:00")` part here?

Reply:
OK, fine by me!

Reply:
It looks like the value used changed from `updatedAt` to `createdAt` in this part of the condition but not above.
Was this intended ?

Reply:
My understanding is that this is intended because the point of the 2nd commit id to "_use the correct timestamp for last_api_call_", i.e. use a different field.

Reply:
yes, it seems for those kinds of logs (anything but 'alerts'), the correct timestamp to take is `createdAt`